library/chemistry/src/Tests.qs

```diff
@@ -316,46 +321,80 @@ operation JordanWignerUCCSTermTest() : Unit {
+    let term1 = [new JordanWignerInputState { Amplitude = NewComplex(2.0, 0.0), FermionIndices = [0, 1, 2, 4] }];
     let state1 = [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,-0.416147, 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,-0.909297, 0., 0., 0.];
     JordanWignerUCCTermTestHelper(5, [0, 1, 3], term1, state1);
 
     // test using Exp(2.0 (a^\dag_1 a^\dag_0 a_2 a_4 - h.c.))
-    let term2 = [new JordanWignerInputState { Amplitude = (2.0, 0.0), FermionIndices = [1, 0, 2, 4] }];
+    let term2 = [new JordanWignerInputState { Amplitude = NewComplex(2.0, 0.0), FermionIndices = [1, 0, 2, 4] }];
     let state2 = [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,-0.416147, 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.909297, 0., 0., 0.];
     JordanWignerUCCTermTestHelper(5, [0, 1, 3], term2, state2);
 
     // test using Exp(2.0 (a^\dag_1 a^\dag_0 a_2 a_4 - h.c.))
-    let term3 = [new JordanWignerInputState { Amplitude = (-2.0, 0.0), FermionIndices = [4, 2, 1, 0] }];
+    let term3 = [new JordanWignerInputState { Amplitude = NewComplex(-2.0, 0.0), FermionIndices = [4, 2, 1, 0] }];
     let state3 = [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,-0.416147, 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.909297, 0., 0., 0.];
     JordanWignerUCCTermTestHelper(5, [0, 1, 3], term2, state2);
 }
 
 @Config(Unrestricted)
 // @Test()
 operation JordanWignerUCCDTermPRQSTest() : Unit {
-    let term0 = [new JordanWignerInputState { Amplitude = (2.0, 0.0), FermionIndices = [2, 0, 4, 1] }];
+    let term0 = [new JordanWignerInputState { Amplitude = NewComplex(2.0, 0.0), FermionIndices = [2, 0, 4, 1] }];
     let state0 = [0., 0., 0., 0., 0.,-0.416147, 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.909297, 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.];
     JordanWignerUCCTermTestHelper(5, [0, 2], term0, state0);
 
-    let term1 = [new JordanWignerInputState { Amplitude = (2.0, 0.0), FermionIndices = [2, 0, 4, 1] }];
+    let term1 = [new JordanWignerInputState { Amplitude = NewComplex(2.0, 0.0), FermionIndices = [2, 0, 4, 1] }];
     let state1 = [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,-0.416147, 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,-0.909297, 0., 0., 0., 0., 0.];
     JordanWignerUCCTermTestHelper(5, [0, 2, 3], term1, state1);
 }
 
 @Config(Unrestricted)
 @Test()
 operation JordanWignerUCCDTermPRSQTest() : Unit {
-    let term3 = [new JordanWignerInputState { Amplitude = (2.0, 0.0), FermionIndices = [0, 4, 2, 3] }];
+    let term3 = [new JordanWignerInputState { Amplitude = NewComplex(2.0, 0.0), FermionIndices = [0, 4, 2, 3] }];
     let state3 = [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.909297, 0., 0., 0., 0.,-0.416147, 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.];
     JordanWignerUCCTermTestHelper(5, [0, 4], term3, state3);
 }
 
+import JordanWigner.JordanWignerClusterOperatorEvolutionSet.ComputeJordanWignerBitString;
+
```

Comment:
Should this get moved up with the other imports or is there a motivation for having it near the tests that use it?

Reply:
I was thinking to put import close to the functions that test that particular import, but it didn't clarify much. I updated to use the usual approach.